<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="ARM_UN">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.000193959779735173 1.26844522706056E-15 -0.0061834838098993"
        rpy="0 0 0" />
      <mass
        value="0.144633575732992" />
      <inertia
        ixx="0.000132553075400075"
        ixy="1.51847330198237E-20"
        ixz="-1.09435411406574E-07"
        iyy="0.000131787795378498"
        iyz="2.21622369370958E-20"
        izz="0.00018833513842507" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="link1">
    <inertial>
      <origin
        xyz="0.0131845085240893 -3.27709857055954E-07 0.00499374991058124"
        rpy="0 0 0" />
      <mass
        value="0.0641795433921742" />
      <inertia
        ixx="3.57552579206163E-05"
        ixy="5.44632634004127E-11"
        ixz="-7.78953701156361E-07"
        iyy="2.43138133345681E-05"
        iyz="1.11829644787619E-11"
        izz="1.84127152791205E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint1"
    type="revolute">
    <origin
      xyz="0 0 0.0596"
      rpy="-3.1416 0 0" />
    <parent
      link="base_link" />
    <child
      link="link1" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-1.5708"
      upper="1.5708"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link2">
    <inertial>
      <origin
        xyz="0.132743929558428 -1.48488428705172E-07 0.00389268768643134"
        rpy="0 0 0" />
      <mass
        value="0.130627170932376" />
      <inertia
        ixx="3.54949904545134E-05"
        ixy="-1.33606183720603E-10"
        ixz="-4.11463070103512E-06"
        iyy="0.000124903135713284"
        iyz="5.01005463977991E-11"
        izz="0.000135802670187777" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint2"
    type="revolute">
    <origin
      xyz="0.02065 0 0"
      rpy="3.1416 1.5708 0" />
    <parent
      link="link1" />
    <child
      link="link2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.5708"
      upper="1.5708"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link3">
    <inertial>
      <origin
        xyz="0.103282270254708 -0.000200370080136847 0.00296707275748816"
        rpy="0 0 0" />
      <mass
        value="0.0972495534277875" />
      <inertia
        ixx="2.14639457040368E-05"
        ixy="1.07686725859436E-07"
        ixz="4.56452510621127E-06"
        iyy="0.000107916212132843"
        iyz="-1.79838781868967E-11"
        izz="0.000107335643249849" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint3"
    type="revolute">
    <origin
      xyz="0.2109 0 0.02065"
      rpy="0 0 -1.5708" />
    <parent
      link="link2" />
    <child
      link="link3" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.5708"
      upper="1.5708"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link4">
    <inertial>
      <origin
        xyz="2.77555756156289E-17 0 0"
        rpy="0 0 0" />
      <mass
        value="5.23598775598299E-07" />
      <inertia
        ixx="5.23598775598299E-14"
        ixy="0"
        ixz="7.00649232162409E-46"
        iyy="5.23598775598299E-14"
        iyz="-4.89097216330415E-46"
        izz="5.23598775598299E-14" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint4"
    type="revolute">
    <origin
      xyz="0.2298 -0.0001 0"
      rpy="-1.5708 0 0" />
    <parent
      link="link3" />
    <child
      link="link4" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-1.5708"
      upper="1.5708"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link5">
    <inertial>
      <origin
        xyz="0 0 2.77555756156289E-17"
        rpy="0 0 0" />
      <mass
        value="5.23598775598299E-07" />
      <inertia
        ixx="5.23598775598299E-14"
        ixy="4.89097216330415E-46"
        ixz="-7.00649232162409E-46"
        iyy="5.23598775598299E-14"
        iyz="0"
        izz="5.23598775598299E-14" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint5"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="0 1.5708 0" />
    <parent
      link="link4" />
    <child
      link="link5" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-1.5708"
      upper="1.5708"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="link6">
    <inertial>
      <origin
        xyz="0 2.77555756156289E-17 0"
        rpy="0 0 0" />
      <mass
        value="5.23598775598299E-07" />
      <inertia
        ixx="5.23598775598299E-14"
        ixy="-7.00649232162409E-46"
        ixz="-4.89097216330415E-46"
        iyy="5.23598775598299E-14"
        iyz="0"
        izz="5.23598775598299E-14" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ARM_UN/meshes/link6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint6"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="1.5708 0 0" />
    <parent
      link="link5" />
    <child
      link="link6" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.5708"
      upper="1.5708"
      effort="0"
      velocity="0" />
  </joint>
</robot>